omod/src/main/webapp/portlets/flagPatientOverview.jsp

```diff
@@ -1,8 +1,22 @@
+<%@page import="org.openmrs.module.patientflags.Flag"%>
 <%@ include file="/WEB-INF/template/include.jsp"%>
 
 <openmrs:require privilege="View Patients" otherwise="/login.htm"
 	redirect="/index.htm" />
 
 <c:forEach items="${model.flags}" var="flag">
-	<span ${flag.priority != null ? flag.priority.style : ''}>${flag.localizedMessage}</span><br />
+	<span ${flag.priority.style}> 

```

Comment:
Links below states that EL suppresses NPE, while JSTL doesnot. The code here wont throw NPE unless JSTL is used
http://www.coderanch.com/t/289665/JSP/java/null-JSTL
http://stackoverflow.com/a/4749093/830945